packages/ttd-cli.vm/ttd-cli.vm.nuspec

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>ttd-cli.vm</id>
```

Comment:
[nitpick] Why not using just the tool name: `ttd.vm`?

Reply:
I think it would be too short for a package name.

Reply:
Doesn't matter which way we go with it for me, but `ttd` alone would be okay imo. We already have other short names like `die`, `c3`, `rla`, `upx`, `map`, `isd`, `file`, `nasm`, `nmap`, etc

Reply:
@emtuls 
Done.